doc/debugging-an-app.md

```diff
@@ -59,7 +60,7 @@ requests.
 Let's figure out where those are coming from. Let's run the `tap` command again,
 and grep the output for `Unknown`s:
 
-####  ```conduit tap deploy emojivoto/voting | grep Unknown -B 2```
+####  ```conduit -n emojivoto tap deploy/voting | grep Unknown -B 2```
 
 ```
 req id=0:212 src=172.17.0.8:58326 dst=172.17.0.10:8080 :method=POST :authority=voting-svc.emojivoto:8080 :path=/emojivoto.v1.VotingService/VotePoop
```

Comment:
would be nice, but not essential, to update the `dst` labels to be pod names in these examples